### DISCUSSION TITLE
win版,有成功找到站点的吗?

### DISCUSSION THREAD

**Discussion by hobinleon** [sunyuting83/v2rss-go#8] [Q&A]
我一个都没有找到,不知道是不是软件失效了,所以问问.
另外,如果找到了,作者能不能再命令行给个提示啊.比如打印出地址,类型等等

**Comment by sunyuting83** [comment]
下载 多订阅爬取 的版本。
提示就不做了，没有意义。win版只是顺便打包的。主要还是部署在vps和软路由上的。
